test/llvm-intrinsics/llvm-vector-reduce/add.ll

```diff
@@ -0,0 +1,552 @@
+; RUN: llvm-as %s -o %t.bc
```

Comment:
Given the amount of effort that has already gone into writing this test, I will categorize this comment as 'feel free to ignore' and 'non-blocking'. Do we want to test the reverse translated LLVM code here?

Thanks

Reply:
IMO. no. These are covered by appropriate instruction tests written in the past - such effort will be redundant when such tests are in place.

Reply:
Hmm. I do not think we have instruction level tests anywhere. Please feel free to point me to such tests.

Thanks